src/assignment/assignment.service.ts

```diff
@@ -1492,16 +1498,30 @@ export class AssignmentService {
 				stringifyUrl({
 					url: `${getEnv('PROXY_URL')}/assignments/${assignmentId}/share/add-contributor`,
 					query: {
-						email: user.email,
-						rights: user.rights,
+						email: invitee.email,
+						rights: invitee.rights,
 					},
 				}),
 				{ method: 'POST' }
 			);
+
+			trackEvents(
+				{
+					object: assignmentId,
+					object_type: 'assignment',
+					action: 'share',
+					resource: {
+						education_level: assignment.education_level_id,
+						rights: invitee.rights || 'NONE',
+					},
+				},
+				inviter
+			);
```

Comment:
check that these events make it into the events database
https://event-logger-qas-meemoo-infra.private.cloud.meemoo.be/console/api/api-explorer

password in 1p

Reply:
Yup, being saved correctly see `a4d791c6-eacc-44cc-9628-74144ca3be24`